### PR TITLE
EventTilePreview fixes

### PR DESCRIPTION
**Pull request by robintown**
Just a couple of things I noticed while referencing EventTilePreview code for #5999. (see commits)

Fixes https://github.com/vector-im/element-web/issues/17621

### CODE REVIEW

**Comment by robintown** [comment]
Also, PR 6000 ;)

File: src/components/views/elements/EventTilePreview.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ interface IState {
     message: string;
 }
 
-const AVATAR_SIZE = 32;
+const AVATAR_SIZE = 30;
```
Comment (turt2live) [inline src/components/views/elements/EventTilePreview.tsx:64]: why?

**Review by turt2live — CHANGES_REQUESTED** [review]
otherwise looks sane, but we do need a comment of some kind for why we'd be dropping to a lower avatar size (risking blurry images)

**Comment by robintown** [comment]
Reverted the thumbnail size change as I wasn't aware that 32 is just the value that's used everywhere. This should just be the one bug fix for when the user doesn't have a display name, then

**Comment by robintown** [comment]
Also removed the mystery dot added by #6154

settings

**Review by turt2live — APPROVED** [review]
Thank you! Apologies for the super delayed turnaround time on this...

**Pull request merged**